kaiax/gasless/config.go

```diff
@@ -26,38 +26,55 @@ var (
 		Name:     "gasless.allowed-tokens",
 		Usage:    "allow token addresses for gasless module, allow all tokens if all",
 		Value:    cli.NewStringSlice("all"),
-		Aliases:  []string{"genesis.module.gasless.allowed-tokens"},
+		Aliases:  []string{"kaiax.module.gasless.allowed-tokens"},
 		Category: "KAIAX",
 	}
 	DisableFlag = &cli.BoolFlag{
 		Name:     "gasless.disable",
 		Usage:    "disable gasless module",
 		Value:    false,
-		Aliases:  []string{"genesis.module.gasless.disable"},
+		Aliases:  []string{"kaiax.module.gasless.disable"},
+		Category: "KAIAX",
+	}
+	MaxGaslessBundleSizeFlag = &cli.IntFlag{
+		Name:     "gasless.max-gasless-bundle-size",
+		Usage:    "max size of gasless bundle, default value if 0, no limit if minus value",
```

Comment:
```suggestion
		Usage:    "max size of gasless bundle. Default value is 100. No limit if negative value",
```

Reply:
https://github.com/urfave/cli/blob/efab65a84b56dd9c80562dbc4e9b7a52b1f97a87/flag_int.go#L49-L50
Flag cannot determine if an argument is the specified zero or empty. So this uses default value even if 0,

Reply:
fixed at https://github.com/kaiachain/kaia/pull/328/commits/a39d6a3a94ee35c50c7f99de0e1eddee12f833b6

Reply:
Then we should fix `Value:    100,` to `Value:    0,`.

Reply:
Yes, default value is 100. GA is using the max value 100 without the flag. This is simply we cannot set 0 to the max value.